packages/next/build/webpack/loaders/next-client-pages-loader.ts

```diff
@@ -13,15 +13,15 @@ const nextClientPagesLoader: loader.Loader = function() {
 
   return `
     (window.__NEXT_P=window.__NEXT_P||[]).push([${stringifiedPage}, function() {
-      var page = require(${stringifiedAbsolutePagePath})
+      var mod = require(${stringifiedAbsolutePagePath})
       if(module.hot) {
         module.hot.accept(${stringifiedAbsolutePagePath}, function() {
           if(!next.router.components[${stringifiedPage}]) return
           var updatedPage = require(${stringifiedAbsolutePagePath})
           next.router.update(${stringifiedPage}, updatedPage.default || updatedPage)
```

Comment:
What about this?

Reply:
What do you mean, do you think this part can also be shortened some?

Reply:
Oh, just wasn't sure if we should change this too since we're handling the `.default` later on.

Reply:
It does look like it would make sense to move this handling to later on also since then it's centralized to there instead of duplicated for each page 👍